apps/full-site-editing/package.json

```diff
@@ -36,6 +36,7 @@
 		"@wordpress/i18n": "^3.3.0",
 		"@wordpress/url": "^2.5.0",
 		"classnames": "^2.2.6",
-		"lodash": "^4.17.11"
+		"lodash": "^4.17.11",
+		"prop-types": "^15.7.2"
```

Comment:
since we've gone down the hooks path, we no longer need this change on `package.json`. 😉